assets/README.md

```diff
@@ -25,4 +26,4 @@ Each file represents an equipment type and is listed with the following attribut
 
 Pre-commit checks will be made on these files to ensure:
 * No duplicated asset numbers - if given these need to be globally unique
-* No duplicated serial numbers - these need to be unique within each equipment make & model set.
+* No duplicated serial numbers - these need to be unique within each equipment make &amp; model set.
```

Comment:
same suggestion as above (`and` instead of `&`)

Reply:
> Hi Mark, that looks good to me. I did some quick "mocked testing" and the tests are nicely capturing errors in metadata entries. Haven't done anything extensive, but looks very good to me.
> 
> Just a minor suggestion inline on the use of plain words.

The problem is that markdown isn't really the same as plain text. Github hides some of the complexities, try typing in "&lt;hello&gt;" into a markdown file.  Using `&amp;` is the safest way, likewise `&lt;` and `&gt;` are needed.  I would consider swapping `&amp;` for "and"  however.

Reply:
oh yes totally agree that we shall use markdown proper syntax when we need. That one was mainly because there was an easy alternative, but, as I said, only a very minor and not necessary change! That happens when everything else is perfect! ;-)

Reply:
ha ha, nice one with the "<hello>" , just tried it!! and now I want to know if there are easter eggs... ;-P